gandy/models/hypersearch.py

```diff
@@ -9,85 +9,88 @@
 
     Typical usage:
 
-    Define `search_space` a dictionary of hyperparameters and their respective 
+    Define `search_space` a dictionary of hyperparameters and their respective
     space to search.
-    
+
     search_space = {'hyp1': [choice1, choice2, choice3],
                     'hyp2': (low, high, uniform)}
-                    
+
     For a set of development data Xs, Ys as arrays with non zeroth dimension
     shapes xshape and yshape:
-    
-    opt = OptRoutine(UncertaintyModel, Xs, Ys, search_space, xshape=xshape, 
+
+    opt = OptRoutine(UncertaintyModel, Xs, Ys, search_space, xshape=xshape,
                      yshape=yshape)
-    
+
     Optimize with 3 fold CV
     opt.optimize(k=3)
     best_params = opt.best_params # dict of the values in search_space for each
-                                  # hyp found to be best 
+                                  # hyp found to be best
 """
 
-## imports
+# imports
 from typing import Tuple, Iterable, Any, Object, Type, List
 
 import optuna
 import numpy
 
 import gandy.models.models
 
-## Typing
+# Typing
 Model = Type[gandy.models.models.UncertaintyModel]
 Array = Type[numpy.ndarray]
 Trial = Type[optuna.trials.Trial]
 
-## class to specify optuna search space from python readable inputs
+# class to specify optuna search space from python readable inputs
+
+
 class SearchableSpace:
-    """Wrapper to convert user specified search space into Optuna readable 
+    """Wrapper to convert user specified search space into Optuna readable
     function.
-    
+
     Args:
-        hypname (str): 
+        hypname (str):
             Name of hyperparameter.
         space (tuple or list):
             The user defined hyperparameter space to search, determined by form
             Options - TBD
-    
+
     Attributes:
-        func (optuna.trials.Trial method): 
+        func (optuna.trials.Trial method):
             Function to be used for sampling of hyperparams.
-        hypname (str): 
+        hypname (str):
             Name of hyperparameter.
         args (tuple): Positional arguments after name to be passed to func for sampling
     """
+
     def __init__(self, hypname, space):
-        ## pseudocode
-        #. if statement format of space
+        # pseudocode
+        # . if statement format of space
         #      set self.func, self.args, and self.hypname
         return
-        
 
-## object function class to be optimized
+
+# object function class to be optimized
```

Comment:
Same for this: You might want to include this comment in the doc string instead of above the class definition.